src/api/json/catalog.json

```diff
@@ -130,6 +165,16 @@
         "galaxy.yml"
       ]
     },
+    {
+      "name": "Ansible-lint Configuration",
+      "description": "Ansible-lint Configuration",
+      "url": "https://raw.githubusercontent.com/ansible-community/schemas/main/f/ansible-lint.json",
+      "fileMatch": [
+        ".ansible-lint",
+        ".config/ansiblelint.yml",
```

Comment:
This is incorrect, linter does not recognize `.config/ansiblelint.yml` as a config file. The other to are correct.